Assets/Scripts/SetValueObjectMesh.cs

```diff
@@ -26,23 +28,20 @@ public class SetValueObjectMesh : CodeBlockObjectMesh {
 
         // Jank fix for text pos, maybe move it to it's codeblock?
         private void UpdateText() {
-            if (textMesh == null) {
-                textMesh = GetComponentInChildren<TextMeshPro>(true);
-            }
-            if (textMesh == null) {
-                GameObject codeBlockTextGameObject = Instantiate(
-                    Resources.Load<GameObject>(ResourcePathConstants.CodeBlockTextPrefab), GetMyCodeBlock().transform) as GameObject;
-                codeBlockTextGameObject.transform.SnapToParent(middle);
-                Vector3 tmp = codeBlockTextGameObject.transform.localPosition;
-                tmp.x = 0;
-                codeBlockTextGameObject.transform.localPosition = tmp;
+            if (textMesh == null && textMeshSetVar == null) {
+                InstantiateText();
+                RepositionText();
                 StartCoroutine(UpdateTextNextFrame());
-            }
-            else {
+                textMeshSetVar = top.GetChild(1).GetComponent<TextMeshPro>();
+                textMesh = middle.GetChild(0).GetComponent<TextMeshPro>();
+            } else {
                 textMesh.SetText("To");
+                textMeshSetVar.SetText("Set\nVar");
                 // Forces Text update
```

Comment:
I actually just made an extension method for this, you can now do `textMesh.ForceTextUpdate()` see `Extensions.cs` for the method